13.roman-to-integer.py

```diff
@@ -0,0 +1,24 @@
+#
+# @lc app=leetcode id=13 lang=python3
+#
+# [13] Roman to Integer
+#
+
+# @lc code=start
+class Solution:
+    def romanToInt(self, s: str) -> int:
+        """
+        time: O(len(s))
+        space: O(1)
+        """
+        ans = 0
+        dict_ = {'I' : 1, 'V' : 5, 'X' : 10, 'L' : 50, 'C' : 100, 'D' : 500, 'M' : 1000,}
```

Comment:
dict_ is better than dict, but it just shows the type of the variable. Change the name to xxx_dict, xxx_hash, xxx_map.
One simple name is roman_dict